apps/user/src/app/signup/page.tsx

```diff
@@ -42,21 +42,21 @@ const SignUpPage = () => {
                                 onChange={handleJoinUserDataChange}
                             />
                             <ButtonInput
-                                label="이메일 인증"
-                                buttonText={isRequestEmail ? '재전송' : '인증'}
+                                label="전화번호 인증"
+                                buttonText={isVerification ? '재전송' : '인증'}
                                 onClick={() => {
-                                    handleRequestEmailButtonClick();
+                                    handleVerificationButtonClick();
                                     startTimer();
                                 }}
                                 enabled={isButtonDisabled}
-                                type="email"
-                                placeholder="이메일"
+                                type="phoneNumber"
+                                placeholder="전화번호"
```

Comment:
```suggestion
                                placeholder="- 없이 입력해주세요"
```

다른 전화번호 입력란처럼 이렇게 수정하는건 어떨까요?